modules/ossm-observability-access.adoc

```diff
@@ -6,13 +6,27 @@
 [id="ossm-observability-access-console_{context}"]
 = Accessing the Kiali console
 
-To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+To access the Kiali console you must have {ProductName} installed and projects configured for the service mesh.
 
-1. In the {product-title} menu bar, click the *Application launcher* > *Kiali*.
+.For administrators
 
-2. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+1. Use the perspective switcher to switch to the Administrator perspective. 
 
-3. Select the project for your service in the *Namespace* field. If you have installed the Bookinfo example, select `bookinfo`.
+2. Click *Home* > *Projects*.
+
+3. In the Launcher section, click `Open on Kiali`.
+
+4. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
+
+.For developers
+
+1. Use the perspective switcher to switch to the Developer perspective. 
+
+2. Click *Topology*.
+
+3. Click `Kiali` from a node in the topology you have configured for the service mesh.
```

Comment:
Kiali link is not in each topology node but only one on toolbar on the right side.
![Screenshot from 2020-12-11 10-43-27](https://user-images.githubusercontent.com/604313/101887873-b9521280-3b9d-11eb-9599-cdc73306a9ce.png)